mscore/prefsdialog.cpp

```diff
@@ -69,6 +69,7 @@ PreferenceDialog::PreferenceDialog(QWidget* parent)
```

Comment:
Could preferencesChanged be called directly instead of being linked with signals?
That way, you could just disable the push bitton when saving starts and reenable it at the end.
It would to be done in another thread though, and i wonder if it's a good idea.